src/main/java/com/glodblock/github/client/gui/base/FCGuiMonitor.java

```diff
@@ -685,12 +684,13 @@ public Enum<?> getSortDisplay() {
     }
 
     @Override
-    public Enum getTypeFilter() {
+    public Enum<?> getTypeFilter() {
         return this.configSrc.getSetting(Settings.TYPE_FILTER);
     }
 
     @Override
-    public void updateSetting(final IConfigManager manager, final Enum settingName, final Enum newValue) {
+    public void updateSetting(final IConfigManager manager, @SuppressWarnings("rawtypes") final Enum settingName,
```

Comment:
Why are these still rawtypes?

Reply:
The super method is part of the AE2 API and uses rawtypes, so the override can't use `<?>`.